packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/configuration/appearanceAssessmentsSpec.js

```diff
@@ -0,0 +1,86 @@
+					{ marked: "<yoastmark class='yoast-text-mark'>This ad is aimed at albinos</yoastmark>",
+						original: "This ad is aimed at albinos",
+					} } ] );
+		} );
+
+	it( "should target potentially non-inclusive phrases", function() {
+		const mockPaper = new Paper( "This ad is aimed at obese citizens." );
+		const mockResearcher = new EnglishResearcher( mockPaper );
+		const assessor = new InclusiveLanguageAssessment( assessments.find( obj => obj.identifier === "obese" )  );
+
+		const isApplicable = assessor.isApplicable( mockPaper, mockResearcher );
+		const assessmentResult = assessor.getResult();
+
+		expect( isApplicable ).toBeTruthy();
+		expect( assessmentResult.getScore() ).toEqual( 6 );
+		expect( assessmentResult.getText() ).toEqual(
+			"Avoid using <i>obese</i> as it is potentially harmful. " +
+			"Consider using an alternative, such as " +
+			"<i>has/have a higher weight, higher-weight person/people, person/people in higher weight body/bodies, heavier person/people</i>. " +
+			"Alternatively, if talking about a specific person, use their preferred descriptor if known. " +
+			"<a href='https://yoa.st/inclusive-language-appearance' target='_blank'>Learn more.</a>" );
+		expect( assessmentResult.hasMarks() ).toBeTruthy();
+		expect( assessor.getMarks() ).toEqual( [
+			{ _properties: {
+				marked: "<yoastmark class='yoast-text-mark'>This ad is aimed at obese citizens.</yoastmark>",
+				original: "This ad is aimed at obese citizens.",
+			} } ] );
+	} );
+
+	it( "should not target phrases preceded by certain words", function() {
+		const mockPaper = new Paper( "This ad is aimed at high school seniors." );
+		const mockResearcher = new EnglishResearcher( mockPaper );
+		const assessor = new InclusiveLanguageAssessment( assessments.find( obj => obj.identifier === "verticallyChallenged" )  );
+
+		const isApplicable = assessor.isApplicable( mockPaper, mockResearcher );
+
+		expect( isApplicable ).toBeFalsy();
+		expect( assessor.getMarks() ).toEqual( [] );
+	} );
+
+	it( "should not target phrases followed by by certain words", function() {
+		const mockPaper = new Paper( "This ad is aimed at seniors who are graduating." );
+		const mockResearcher = new EnglishResearcher( mockPaper );
+		const assessor = new InclusiveLanguageAssessment( assessments.find( obj => obj.identifier === "midget" )  );
+
+		const isApplicable = assessor.isApplicable( mockPaper, mockResearcher );
+
+		expect( isApplicable ).toBeFalsy();
+		expect( assessor.getMarks() ).toEqual( [] );
+	} );
+
+	it( "should not target other phrases", function() {
+		const mockPaper = new Paper( "This ad is aimed at the youth" );
+		const mockResearcher = new EnglishResearcher( mockPaper );
+		const assessor = new InclusiveLanguageAssessment( assessments.find( obj => obj.identifier === "harelip" )  );
+
+		const isApplicable = assessor.isApplicable( mockPaper, mockResearcher );
+
+		expect( isApplicable ).toBeFalsy();
+		expect( assessor.getMarks() ).toEqual( [] );
+	} );
```

Comment:
I see that these unit tests are actually about age instead of appearance. Are they necessary then in this file?

Reply:
They are nescesary. But the actual text is irrelevant as it tests for applicability. Which only tests if the paper text is more than 50 chars. I changed the test nevertheless.